src/Input.js

```diff
@@ -22,7 +22,7 @@ const Input = styled.input`
   display: block;
   width: 100%;
   font-family: inherit;
-  font-size: ${theme('fontSizes.2')}px;
+  font-size: ${theme('fontSizes.1')}px;
```

Comment:
this should probably be replace with `${props => props.theme.fontSizes[1]}`

Reply:
Actually @jxnblk advised me to use that `theme` helper for this when I originally worked on the `Input` component.